modules/core/src/main/java/org/apache/ignite/spi/systemview/view/SnapshotView.java

```diff
@@ -143,6 +147,14 @@ public String type() {
         return type.name();
     }
 
+    /**
+     *  @return Creation timestamp in milliseconds since Unix epoch.
+     */
+    @Order(7)
+    public Long creationTimeMillis() {
+        return creationTimeMillis != null && creationTimeMillis != 0 ? creationTimeMillis : null;
```

Comment:
it never can be null.

Reply:
it can be null in case of backward compatibility. when SnapshotView tries to output outdated snapshot without creationTime metadata this field stores zero value and it should be prints as null